spec/models/visit_spec.rb

```diff
@@ -343,4 +343,40 @@
       end
     end
   end
+
+  describe '#allowed_additional_visitors' do
+    let(:visitor1) { FactoryGirl.build_stubbed(:visitor) }
```

Comment:
no need to add the `FactoryGirl` just use the short syntax, it looks a lot less cluttered and we don't have any other conflicting stubbing library.